flaskbb/plugins/__init__.py

```diff
@@ -10,8 +10,35 @@
 """
 from flask import current_app
 from flask_plugins import Plugin
-
+from flask_migrate import upgrade,downgrade,migrate
+from flaskbb.extensions import db,migrate as mig
 from flaskbb.management.models import SettingsGroup
+from flask import g
+import contextlib
+import os,copy
+
+@contextlib.contextmanager
+def plugin_name_migrate(name):
```

Comment:
could you fix the pep8 erros and add some comments/docstrings in the next 3 methods (plugin_name_migrate, db_for_plugin, config_migrate)? My editor automatically highlights the pep8 errors.